.gitignore

```diff
@@ -26,3 +26,4 @@ node_modules/
 .tox
 dist/
 site/
+.env
```

Comment:
You mean that the ignore should not be present, or that a separate PR should be made for this change only?